.env

```diff
@@ -0,0 +1,2 @@
+MONGOHQ_URL=mongodb://heroku:c-rJompa5mbNDiERpkBJMfKXbfSazY_YaBGqrnFzmwgao-WRaPHaU_ZED_TU7zp2C8lp3OV1azqFINmN6EQ27A@candidate.37.mongolayer.com:10928,candidate.13.mongolayer.com:11150/app40069500
```

Comment:
This looks like it might contain an API key. If that's the case, you need to revoke the key ASAP. There are bots that scrape GitHub looking for keys and using them to mine Bitcoin and run botnets and other bad things. Usually, it's with AWS, but I imagine they can abuse heroku, too.

Even if it's not, you should add any config and environment stuff to your .gitignore. It shouldn't be committed to the repo.